plugins/plugin_utils/file_utils.py

```diff
@@ -0,0 +1,88 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import errno
+import os
+
+from ansible.utils.display import Display
+from ansible_collections.azure.azcollection.plugins.plugin_utils import constants as consts
+from ansible.errors import AnsibleParserError, AnsibleError
```

Comment:
Unused AnsibleParserError imported from ansible.errors

```suggestion
from ansible.errors import AnsibleError
```